src/_data/sidenav.yml

```diff
@@ -389,7 +389,7 @@
       permalink: /deployment/windows
     - title: Build and release a web app
       permalink: /deployment/web
-    - title: Continuous deployment
+    - title: Setup continuous deployment
```

Comment:
**issue:** This is `set up` when used as a verb per [GSG](https://developers.google.com/style/word-list#setup).
```suggestion
    - title: Set up continuous deployment
```